packages/advanced-logic/specs/payment-network-erc20-fee-proxy-contract-0.1.0.md

```diff
@@ -43,6 +35,7 @@ The contract contains one function called `transferFromWithReferenceAndFee` whic
 
 The `TransferWithReferenceAndFee` event is emitted when the tokens are transfered. This event contains the same 6 arguments as the `transferFromWithReferenceAndFee` function.
 
+TODO: don't merge without fixing this TODO
 TODO: [See smart contract source]()
```

Comment:
Oups, forgot this one. Here it goes:
```suggestion
[See smart contract source](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/smart-contracts/src/contracts/ERC20FeeProxy.sol)
```

Reply:
Thanks, as you can see, I forgot as well :) :)